internal/internal_task_pollers.go

```diff
@@ -464,15 +464,44 @@ func (lath *localActivityTaskHandler) executeLocalActivityTask(task *localActivi
 		rootCtx = context.Background()
 	}
 
+	timeoutDuration := time.Duration(task.params.ScheduleToCloseTimeoutSeconds) * time.Second
+	deadline := time.Now().Add(timeoutDuration)
+	if task.attempt > 0 && !task.expireTime.IsZero() && task.expireTime.Before(deadline) {
+		// this is attempt and expire time is before SCHEDULE_TO_CLOSE timeout
+		deadline = task.expireTime
+	}
+
+	var scheduled time.Time
+	var started time.Time
+	var taskToken []byte
+	if task.workflowTask != nil && task.workflowTask.task != nil {
+		scheduled = time.Unix(0, task.workflowTask.task.GetScheduledTimestamp())
+	}
+
+	if task.workflowTask != nil && task.workflowTask.task != nil {
+		started = time.Unix(0, task.workflowTask.task.GetStartedTimestamp())
+	}
+
+	if task.workflowTask != nil && task.workflowTask.task != nil {
+		taskToken = task.workflowTask.task.TaskToken
+	}
```

Comment:
this doesn't make sense to me. For local activity task, these things don't really exist or mean anything